### DISCUSSION TITLE
Structured Outputs支持有问题

### DISCUSSION THREAD

**Discussion by adriftliu** [BurnHair/docs#193] [技术支持]
gpt-4o-mini-2024-07-18 理论上是支持[Structured Outputs](https://platform.openai.com/docs/guides/structured-outputs/structured-outputs)
请求你这的pydantic就报错，另外一家相同模型就可以。

**Comment by BennyThink** [comment]
暂时先试试 https://api.burn.hair/ 吧，一直没时间看这边……

**Comment by BennyThink** [comment]
修好啦！

**Reply by adriftliu** [reply]
谢谢 有空试一下
